src/base/rss/rss_autodownloader.cpp

```diff
@@ -192,7 +190,7 @@ void AutoDownloader::handleTorrentDownloadFinished(const QString &url)
     if (!job) return;
 
     if (auto feed = Session::instance()->feedByURL(job->feedURL))
-        if (auto article = feed->articleByGUID(job->articleGUID))
+        if (auto article = feed->articleByGUID(job->articleData.value(Article::KeyId).toString()))
```

Comment:
It's not apparent to know what type `articleByGUID` returns. I would state `Article *` in place of `auto`, this also applies to previous `if (auto feed = ...`